tests/unit/services/cooperation-service.spec.jsx

```diff
@@ -0,0 +1,142 @@
+import {
+  cooperationService,
+  CooperationNotesService
+} from '~/services/cooperation-service'
+import { mockAxiosClient } from '~tests/test-utils'
+import { URLs } from '~/constants/request'
+
+const id = '64kf41f7806a06c65338c509'
+const lessonId = '64ef41f7806a06c65338c433'
+const noteId = '64dc92f7806a06c65338b712'
+
+const creationData = {
+  offer: '64ef92a7806a06c65338d123',
+  receiver: '64bf81c7806a06c65338f456',
+  receiverRole: 'tutor'
+}
+
+const updateResourceStatus = {
+  id,
+  resourceId: lessonId,
+  completionStatus: 'completed'
+}
+
+const noteData = {
+  isPrivate: false,
+  text: 'This is  Note'
+}
+
+describe('Cooperation Service tests', () => {
+  afterEach(() => {
+    mockAxiosClient.resetHistory()
+    mockAxiosClient.reset()
+  })
+
+  it('should create new cooperation', async () => {
+    mockAxiosClient.onPost(new RegExp(URLs.cooperations.create)).reply(200)
```

Comment:
Please, check if it works without `RegExp`

```suggestion
    mockAxiosClient.onPost(URLs.cooperations.create).reply(200)
```

Reply:
fixed